src/pages/EventDetails/EventDetails.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Header from '../../components/Header/Header';
 
-const EventDetails = () => {
+const propTypes = {
+  match: PropTypes.object.isRequired
```

Comment:
matches what?

Reply:
You're correct, my implementation is incomplete. It should look like: 

```
match: PropTypes.shape({
  params: PropTypes.shape({
    id: PropTypes.node,
  }).isRequired,
}).isRequired
```